src/content/protocols/compound-v3-arbitrum.md

```diff
@@ -0,0 +1,219 @@
+| TimeLock                      | queueTransaction                      | Queues a transaction that can be executed once a delay has passed. The current delay is 2 days. This can impact the own TimeLock's settings (change admin, set delays) or interaction with any other contract the DAO has permissions on.                                                                                                                                                                                                                 | ArbitrumBridgeReceiver (Mainnet DAO)            |
+| TimeLock                      | cancelTransaction                     | Cancels a pending transaction and removes it from the queue. This allows the DAO to cancel one of its own decision before it is executed.                                                                                                                                                                                                                                                                                                                 | ArbitrumBridgeReceiver (Mainnet DAO)            |
+| TimeLock                      | executeTransaction                    | Executes a transaction that was previously queued, if the corresponding delay has passed.                                                                                                                                                                                                                                                                                                                                                                 | Governor (DAO)                                  |
+
+The permissions for all Comet contracts (USDC, WETH, wsETH, USDT, USDS) are similar and therefore only
+represented once as `Comet Proxy` and `Comet Implementation`in the table above.
+
+## Dependencies
+
+The compound-v3 protocol relies on a Chainlink oracle feed to price collateral and base assets in the system. The protocol does not validate asset prices returned by Chainlink feeds other than checking for a zero-value. The protocol further does not offer a fallback pricing mechanism in case the Chainlink oracle feeds are stale or untrusted. If not performing as expected, Chainlink oracle feeds can only be replaced through a regular Compound governance proposal with a delay (see Exit Window).
+
+The Chainlink oracle system itself is upgradeable potentially resulting in the publishing of unintended or malicious prices. The permissions to upgrade are controlled by a [multisig account](https://etherscan.io/address/0x21f73D42Eb58Ba49dDB685dc29D3bF5c0f0373CA) with a 4-of-9 signers threshold. This multisig account is listed in the Chainlink docs but signers are not publicly announced. The Chainlink multisig thus does not suffice the Security Council requirements specified by either L2Beat or DeFiScan resulting in a High centralization score.
+
+## Upgrade process
```

Comment:
reflects the Mainnet upgrade process, on Arbitrum reflect cross-chain messaging here